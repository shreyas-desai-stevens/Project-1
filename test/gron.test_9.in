{
    "abc": 
    {
        "id": 1,
        "value": "File",
        "popup": 
        {
            "menuitem": 
            [
                {"value": "New", "onclick": "CreateNewDoc()"},
                {"value": "Open", "onclick": "OpenDoc()"},
                {"value": "Close", "onclick": "CloseDoc()"}
            ]
        }
    },
    "ab": 
    {
        "id": 2,
        "value": "File",
        "popup": 
        {
            "menuitem": 
            [
                {"value": "New", "onclick": "CreateNewDoc()"},
                {"value": "Open", "onclick": "OpenDoc()"},
                {"value": "Close", "onclick": "CloseDoc()"}
            ]
        }
    },
    "a": 
    {
        "id": 3,
        "value": "File",
        "popup": 
        {
            "menuitem": 
            [
                {"value": "New", "onclick": "CreateNewDoc()"},
                {"value": "Open", "onclick": "OpenDoc()"},
                {"value": "Close", "onclick": "CloseDoc()"}
            ]
        }
    }
}