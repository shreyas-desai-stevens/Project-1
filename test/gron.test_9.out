json = {};
json.a = {};
json.a.id = 3;
json.a.popup = {};
json.a.popup.menuitem = [];
json.a.popup.menuitem[0] = {};
json.a.popup.menuitem[0].onclick = "CreateNewDoc()";
json.a.popup.menuitem[0].value = "New";
json.a.popup.menuitem[1] = {};
json.a.popup.menuitem[1].onclick = "OpenDoc()";
json.a.popup.menuitem[1].value = "Open";
json.a.popup.menuitem[2] = {};
json.a.popup.menuitem[2].onclick = "CloseDoc()";
json.a.popup.menuitem[2].value = "Close";
json.a.value = "File";
json.ab = {};
json.ab.id = 2;
json.ab.popup = {};
json.ab.popup.menuitem = [];
json.ab.popup.menuitem[0] = {};
json.ab.popup.menuitem[0].onclick = "CreateNewDoc()";
json.ab.popup.menuitem[0].value = "New";
json.ab.popup.menuitem[1] = {};
json.ab.popup.menuitem[1].onclick = "OpenDoc()";
json.ab.popup.menuitem[1].value = "Open";
json.ab.popup.menuitem[2] = {};
json.ab.popup.menuitem[2].onclick = "CloseDoc()";
json.ab.popup.menuitem[2].value = "Close";
json.ab.value = "File";
json.abc = {};
json.abc.id = 1;
json.abc.popup = {};
json.abc.popup.menuitem = [];
json.abc.popup.menuitem[0] = {};
json.abc.popup.menuitem[0].onclick = "CreateNewDoc()";
json.abc.popup.menuitem[0].value = "New";
json.abc.popup.menuitem[1] = {};
json.abc.popup.menuitem[1].onclick = "OpenDoc()";
json.abc.popup.menuitem[1].value = "Open";
json.abc.popup.menuitem[2] = {};
json.abc.popup.menuitem[2].onclick = "CloseDoc()";
json.abc.popup.menuitem[2].value = "Close";
json.abc.value = "File";
